tests/bluetooth/bsim/nrf_auraconfig/tests/usecase_0_test.sh

```diff
@@ -9,6 +9,34 @@ SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 SCRIPT_NAME=$(basename "$0")
 
 ${SCRIPT_DIR}/../_nrf_auraconfig_simulation.sh ${SCRIPT_NAME} \
-  "nac" "usecase" "0"\
+  "nac" "preset" "24_2_1" "0"\
```

Comment:
The nac_test will never operate on multiple BIGs, hence one of the 0s missing. Could probably spend some time changing the order of the shell commands so that they look more alike though.